src/components/LeftMenu.jsx

```diff
@@ -0,0 +1,38 @@
+//contains all the leftmenu information and images
+import homeImg from "../assets/home.svg";
+import profileImg from "../assets/profile.svg";
+import { Link } from "react-router-dom";
+
+function LeftMenu() {
+  return (
+    <>
+      <div className="left-menu">
+          <div className="whole-logo">
+            <img
+              className="home-logo"
+              src={homeImg}
+              width={40}
+              alt="home logo"/>
```

Comment:
I would have expected this `img` to be inside of the `Link` component below it. I assumed the links weren't functioning because of this.